provisioning_client/tests/hsm_client_riot_ut/CMakeLists.txt

```diff
@@ -23,3 +23,23 @@ set(${theseTestsName}_h_files
 )
 
 build_c_test_artifacts(${theseTestsName} ON "tests/azure_prov_device_tests")
+
+#Set RIoT ECDH Curve & RIOTBUILD
+target_compile_definitions(${theseTestsName}_exe
+    PUBLIC
+        RIOTBUILD
+        RIOTSECP256R1
+    )
+if(WIN32)
+    target_compile_definitions(${theseTestsName}_dll
+        PUBLIC
+            RIOTBUILD
+            RIOTSECP256R1
+    )
+    target_compile_definitions(${theseTestsName}_testsonly_lib
+        PUBLIC
+            RIOTBUILD
+            RIOTSECP256R1
+    )
+ENDIF(WIN32)
+    
```

Comment:
Nit: here and everywhere if it's missing: new-line at the end of file.

Reply:
Fixed